public/index.html

```diff
@@ -95,16 +222,20 @@
     <script>
       const urlAddress = window.location.href
       const teamId = urlAddress.substring(urlAddress.lastIndexOf('/') + 1)
-      var socket = io()
+      //var socket = io()
```

Comment:
I just commented it out for the time being since it was giving me errors since it wasnt fully implemented, I wanted to focus on getting the core functionalities working first and then get the sockets to work well

Reply:
@gmottn in that case, I think we should make sure the socket.io are working before implementing your changes in the original source codes. I have just realized that without the socket.io, not only the team mode, the individual mode will not work either, since there's no connection between the backend and the frontend to update the scores of the users. We can work together to have the socket.io work. Can you show me what errors you were having?

Reply:
maybe the error you are having is local. That is to say, deleting the folder node-modules and running `npm install` again should do the trick.

Reply:
I will try that out, thanks. Ill let you know if that works

Reply:
Oh also, I figured out a simple way to get a person who joins the game's name... do we still have that functionality on the Welcome page? it might not be needed anymore

Reply:
let's consider changing the Welcome page in another PR. This will simplify the process I believe